spec/requests/cms/cms_page_spec.rb

```diff
@@ -34,17 +34,6 @@
         expect { get '/eggs' }.to raise_error(ActiveRecord::RecordNotFound)
       end
     end
-
-    context 'with a nested page' do
-      before do
-        stub_nested_cms_page
-        get '/subject-practitioners/primary'
-      end
-
-      it 'has the expected class' do
-        expect(assigns(:style_slug)).to eq('subject-practitioners')
-      end
-    end
```

Comment:
I think we still need this example: unless we are doing away with support for nested pages. As above.